Cargo.toml

```diff
@@ -13,6 +13,7 @@ edition = "2021"
 
 [features]
 debug_render = ["gb-lcd/debug_render"]
+registers_logs = ["gb-cpu/registers_logs"]
```

Comment:
running cargo with `--feature gb-cpu/registers_logs` don't work ?

Reply:
It might,  I was just copying the behaviour of the already implemented "features"

Reply:
Should I change it ?

Reply:
no, but it work :)